loopchain/app/app.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+
+class App:
```

Comment:
Do you use 'App' for name of new engine?
This name is written by viewpoint in LFT library but we need another name for loopchain.
I suggest new name 'BlockchainService' or 'LoopchainService_v3'.
How do you think about this name?

Reply:
This PR is for separating loopchain into new structure and old structure.
We have to decide the name on next PR.